client/app/components/manager-navbar/manager-navbar.html

```diff
@@ -0,0 +1,144 @@
+<!-- Navbar -->
+<nav class="oui-navbar oui-navbar_fixed" role="navigation">
```

Comment:
In the majority of cases setting an ARIA role and/or aria-* attribute that matches the default implicit ARIA semantics is unnecessary and not recommended as these properties are already set by the browser.

Reply:
Ok, I take example from the bootstrap navbar for this part.
It's removed